js/engine/dune

```diff
@@ -7,7 +7,9 @@
   libyaml_stubs_js
   yaml_ctypes_overrides
   semgrep.running
+  semgrep.parsing_languages
   tree-sitter.run
+  alcotest
```

Comment:
but you can probably make a js/engine/tests/
separate dir and there go wild.

Reply:
yea thats what I was thinking